analytics/migrations/0009_remove_messages_to_stream_stat.py

```diff
@@ -1,5 +1,5 @@
 from django.db import migrations
-from django.db.backends.postgresql.schema import BaseDatabaseSchemaEditor
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
```

Comment:
Hmm, will new migrations have a problem when importing from the other path?

Reply:
These are all used in type annotations for `RunPython` callbacks, so I’m pretty sure they’re part of a long copy+paste chain and not generated automatically.

Reply:
Oh, no, because it's only the RunPython ones that need to import those, and those get written by copy/paste, not `manage makemigrations`.

Reply:
(My comment got posted late since I got pulled AFK in the middle of posting it).